components/messageBox/messageBox.js

```diff
@@ -1,19 +1,30 @@
 import React, { PropTypes } from 'react';
 import { getClassNamesWithMods, getDataAttributes } from '../_helpers';
 
-const MessageBox = (props) => {
-  const { children, dataAttrs, isError } = props;
-  const mods = props.mods ? props.mods.slice() : [];
+const MessageBox = ({ children, dataAttrs, icon, mods, title, type }) => {
+  const className = getClassNamesWithMods('ui-messageBox', [...mods, type]);
 
-  if (isError) {
-    mods.push(`error`);
-  }
+  const logo = icon && (
+    <div className="ui-messageBox__content__icon">
```

Comment:
I'm pretty sure this naming is not fit to the BEM convension. It doesn't allow to use `__` separator twice in one class name. Should be smth like `ui-messageBox__content-icon`. Please fix it here and in all other places...